feat: app event

r4zendev opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced refreshed button styles with two new variants.
  - Added a welcome modal that greets new users.
  - Rolled out intuitive demo access cards and updated report creation dialogs to enhance the trial experience.
  - Redesigned merchant monitoring views (now “Web Presence”) with improved navigation and report summaries.
  - Added a new `MerchantsStatsCard` component for displaying merchant statistics.
  - Introduced conditional rendering for the "Unresolved Monitoring Alerts" section based on merchant monitoring status.
  - New component `BusinessReportOptionsDropdown` for managing report options.
  - Added a new `AppSidebar` component for enhanced sidebar navigation.
  - Introduced `NavFooter` and `NavMain` components for improved layout and user interaction.

- **UI Enhancements**
  - Updated sidebar navigation and layout for a smoother authenticated experience.
  - Improved styling with new theme colors, refined search input dimensions, and better responsiveness.
  - Adjusted sidebar widths for different screen sizes to enhance usability.
  - Enhanced `MerchantMonitoringReportStatus` component with customizable class names.
  - Refactored `MerchantMonitoringStatusBadge` for improved styling logic and simplified implementation.
  - Simplified the `MerchantMonitoringTable` component to better accommodate demo account statuses.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

r4zendev committed:
feat: create report modal

r4zendev committed:
wip demo details

r4zendev committed:
fixes

r4zendev committed:
fixes

r4zendev committed:
wip

r4zendev committed:
wip

r4zendev committed:
wip

r4zendev committed:
wip

r4zendev committed:
adaptive credibility

r4zendev committed:
adaptive credibility

r4zendev committed:
wip use-report-sections

r4zendev committed:
wip use-report-sections

r4zendev committed:
scrollable report sections

r4zendev committed:
scrollable report sections

r4zendev committed:
wip

r4zendev committed:
feat: better line of business risk indicators visualization

r4zendev committed:
fix: remove redundant code

r4zendev committed:
fix: remove redundant code

r4zendev committed:
demo customer type

r4zendev committed:
fix: imports

r4zendev committed:
fix: imports

r4zendev committed:
fix

r4zendev committed:
fix

r4zendev committed:
CTA banners

r4zendev committed:
fix

r4zendev committed:
fix

r4zendev committed:
fix

r4zendev committed:
welcome modal

alonp99 committed:
chore(release): update packages for demo-200225

- Bump multiple packages to demo-200225 versions
- Include patch updates across various modules
- Update changelogs and package files accordingly

shanegrouber committed:
Bal 3611 - UI changes (#3094)

* refactor(CaseOverview): improve tab generation logic

- Replace inline tab logic with a dedicated function
- Ensure 'ubos' case returns the correct tab name
- Maintain existing functionality for other domain tab names

* refactor(CaseOverview): simplify tab generation logic

- Remove unnecessary getTab function
- Use camelCase on the domain string directly

* feat(MerchantMonitoringBusinessReport): update button label for navigation

- Change button text from "Back" to "View All Reports"
- Improve clarity on navigation purpose for users

* feat(scripts): enhance auto-commit script with specific target file option

- Add command line argument to specify target file for git diff
- Update git commit command to include the target file in commit

* feat(MerchantMonitoring): add conditional alert column to the table

- Introduce 'showAlertColumn' prop to toggle alert column visibility
- Modify MerchantMonitoringTable and related components accordingly
- Update logic to filter columns based on 'showAlertColumn' setting

* feat(MerchantMonitoring): update merchant monitoring logic to include demo flag

- Replace showAlertColumn with isDemo in merchant monitoring logic
- Modify MerchantMonitoringTable to conditionally render alert column based on isDemo
- Adapt useColumns to use isDemo for column visibility control

* refactor(MerchantMonitoring): rename isDemo to isDemoAccount

- Update variable and prop names for clarity
- Ensure consistent naming across components and hooks

* style(sidebar): improve class ordering for consistency

- Reorder class names in Sidebar components for better readability
- Make minor formatting adjustments across several Sidebar and TextEditor components
- Maintain consistent styling practices throughout the codebase

* refactor(report-tabs): streamline report tab configurations

- Remove unnecessary social media and ecosystem analysis tabs
- Simplify the report tabs structure for better clarity
- Ensure the remaining tabs maintain functional integrity

* feat(ui): enhance merchant monitoring for demo accounts

- Add tooltip messages indicating feature limitations for demo accounts
- Disable monitoring action for trial users with appropriate feedback
- Update logic to check account type before rendering specific options

* fix(MerchantMonitoring): update button icons and styles in MerchantMonitoring page

- Replace Table2 icon with Layers for better clarity
- Update button text for Upload Multiple Merchants to Batch Actions
- Refactor button styles to improve visibility and user experience

* feat(MerchantMonitoringBusinessReport): align Options dropdown with website domain

- Implement dropdown menu for monitoring options
- Add confirmation dialog for deboarding merchants
- Integrate feedback mechanism via form fields for user reasons

* refactor(merchant-monitoring): restructure merchant monitoring components

- Convert columns definition to a memoized hook for performance enhancement
- Standardize access to demo account configuration across components
- Improve button label for better clarity on navigation functionality

---------

Co-authored-by: Alon Peretz <8467965+alonp99@users.noreply.github.com>

r4zendev committed:
feat: new navbar

r4zendev committed:
fix: hide monitoring cards for users with disabled ongoing monitoring

r4zendev committed:
feat: content violation column

r4zendev committed:
fix

r4zendev committed:
Merge branch 'bal3550b' into epic/app-event

r4zendev committed:
Merge remote-tracking branch 'origin/bal3550b' into epic/app-event

r4zendev committed:
fix

r4zendev committed:
fixes

r4zendev committed:
center monitored column

shanegrouber committed:
feat(business-reports): add isExample flag to BusinessReportSchema (#3098)

- Include isExample property in the transformed report data
- Update MerchantMonitoringTable to display example badge when applicable
- Update subproject commit for workflows service data migrations

r4zendev committed:
fix avatar placement on collapsed sidebar

r4zendev committed:
add more margin on larger screens

r4zendev committed:
alignment fixes

r4zendev committed:
fix: ui tweaks

r4zendev committed:
re-add conditional render for isAlert column

r4zendev committed:
feat: warning for mobile devices

r4zendev committed:
fix: add captions for each premium feature

r4zendev committed:
fix mobile warning feature, styling tweaks for sidebar

shanegrouber committed:
feat(merchant-monitoring): add correlationId and merchantId display (#3106)

- Introduce correlationId field to the business schema
- Implement merchant ID column in MerchantMonitoringTable
- Include copy-to-clipboard functionality for merchant ID

r4zendev committed:
fix separator showing for non-demo, fix logout button hidden for demo, other small fixes

r4zendev committed:
hardcode calendly link

r4zendev committed:
show tooltip with nav item title, show title in hovercard for premium items

r4zendev committed:
fix: sidebar sticky, small fixes

r4zendev committed:
fix: report page scroll to top automatically

tomer-shvadron committed:
feat(backoffice-v2): update dependencies and add new components

- Add BusinessReportSectionsObserver for report navigation
- Introduce BusinessReportOptionsDropdown for report options
- Include hooks for PDF generation and report reference management

tomer-shvadron committed:
feat(button): add status variant to button styles

- Introduce new 'status' variant for enhanced button customization
- Modify Button component and related styles for better usability
- Update MerchantMonitoring components to support the new variant

r4zendev committed:
fixes

r4zendev committed:
more fixes

r4zendev committed:
dont include example reports in count

r4zendev committed:
add backend validation for demo account report creation

r4zendev committed:
restore package versions

r4zendev committed:
Merge remote-tracking branch 'origin/dev' into epic/app-event

r4zendev committed:
resolve remaining conflicts

r4zendev committed:
remove versions [2]

r4zendev committed:
fixes based on self-review

alonp99 committed:
chore(release): bump versions and update dependencies

- Bump versions across various packages, including common, ui, and workflow-sdk
- Update dependencies to their latest compatible versions, improving stability and features
- Ensure consistency in versioning to facilitate smoother upgrades in future releases

r4zendev committed:
fix

r4zendev committed:
Merge remote-tracking branch 'origin/bal-3462' into epic/app-event

r4zendev committed:
move full onboarding in sidebar

r4zendev committed:
bugfix

Commit:
Merge branch 'dev' into epic/app-event

r4zendev committed:
augment meta type

alonp99 commented on apps/backoffice-v2/src/common/components/molecules/DemoAccessCards/GetFullAccessCard.tsx:
```diff
@@ -0,0 +1,51 @@
+        </p>
+
+        <Button asChild variant="wp-primary" className="justify-start space-x-2 text-base">
+          <a href={BALLERINE_CALENDLY_LINK} target="_blank" rel="noreferrer">
```
You suggest hardcoding it, i agree

r4zendev committed:
bugfix

alonp99 committed:
feat(merchant-monitoring): enhance company name display with example badge

- Add badge to indicate example accounts in the company name column
- Wrap company name in a flex container for improved layout

alonp99 committed:
fix(tests): comment out async rule execution test

- Comment out the asynchronous rule execution test case
- Ensure the test suite continues to run without failures
- No changes were made to the functionality of the rule engine

r4zendev committed:
bugfix
